Helper/Cache/CacheHelper.php

```diff
@@ -53,4 +55,20 @@ public function save(CacheItem $item, array $data)
 
         $this->cache->save($item);
     }
+
+    public static function setCacheHeaders(Request $request, Response $response): Response
+    {
+        if (is_string($response->getContent())) {
+            $response->setCache([
+                'etag' => \md5($response->getContent()),
+                'max_age' => 600,
+                's_maxage' => 3600,
+                'public' => true,
+                'private' => false,
+            ]);
+            $response->isNotModified($request);
+        }
+
+        return $response;
+    }
```

Comment:
And That would make reduce the complexity of the function as follows:

```suggestion
    public static function makeResponseCacheable(Request $request, Response $response): void
    {
        if(! is_string($response->getContent()) {
            return;
        }
        
        $response->setCache([
            'etag' => \md5($response->getContent()),
            'max_age' => 600,
            's_maxage' => 3600,
            'public' => true,
            'private' => false,
        ]);
        $response->isNotModified($request);
    }
```

Reply:
👍

Reply:
c10e0420020120adddea507d70b182cfd0bbd7b0